### DISCUSSION TITLE
About Export of DataTable

### DISCUSSION THREAD

**Discussion by lk94312** [datapane/datapane#128] [Help] [answered]
Hello developer!
Why does the export button of my datatable no longer exist, and I can't download csv anymore.
I found this problem in version 0.11.9, I don’t know if it’s just me

**Comment by mands** [comment, marked as answer]
Hi there,

Exporting files from a `DataTable` as CSV/Excel documents is supported for reports uploaded to a Datapane Server (such as https://datapane.com), but isn't available for saved reports having changed our internal `DataTable` component. However we're looking into ways we can support CSV downloads again on locally saved reports using the new component.

HTH!
